src/Microsoft.OpenApi.Hidi/OpenApiService.cs

```diff
@@ -22,7 +22,7 @@ static class OpenApiService
             string input,
             FileInfo output,
             OpenApiSpecVersion version,
-            OpenApiFormat format,
```

Comment:
Try to use OpenApiFormat?  (nullable OpenApiFormat) so that you can distinguish between being sent a parameter or not.

Reply:
@darrelmiller that's a good suggestion. I adopted it. Thank you.